recipe/meta.yaml

```diff
@@ -20,10 +19,9 @@ requirements:
     - setuptools
     - pip
     - wheel
-
   run:
     - python
-    - aiobotocore 2.4.*
+    - aiobotocore 2.4.2
```

Comment:
The range should be used because this is what is defined by upstream.

Moreover, what matters is not only what is currently released, but also the promise of what will be released in the future (i.e. the promise that aiobotocore will not introduce breaking changes between minor versions).

Reply:
Thanks for clarification.